system_setup/server/controllers/locale.py

```diff
@@ -30,3 +32,65 @@ def start(self):
                           .format(self.model.selected_language))
 
         super().start()
+
+    async def _run_locale_support_cmds(self, lang: str):
+        """ Final commands to run when a valid language is POST'ed. """
+        cmds = [["locale-gen"],
+                ["update-locale", "LANG={}".format(lang)]]
+        env = os.environ.copy()
+
+        cp = await arun_command(["check-language-support", "-l", lang[0:5]],
+                                env=env)
+        if cp.returncode == 0:
+            packages = cp.stdout.strip('\n').split(' ')
+            cmds.append(["apt", "install", "-y"] + packages)
+
+        if self.app.opts.dry_run:
+            for cmd in cmds:
+                log.debug('Would run: ' + ' '.join(cmd))
+        else:
+            for cmd in cmds:
+                cp = await arun_command(cmd, env=env)
+                if cp.returncode:
```

Comment:
Maybe that’s the Go developer in me, but let’s try to not rely on truthy (!= 0) implicit conversation

Reply:
I'll get used to that. I agree that being explicit is always good. It's a bit annoying when you may have False or None, but this one is not the case.